### ISSUE TITLE
Hibernation: wake up in Safari not removable

### ISSUE THREAD

**Issue by burkardendres** [gardener/dashboard#1427]
**What happened**:
I tried to configure a cluster in **Safari** to not having a hibernation wake up configured, but an hibernation, but it was always using 12:30 as a default, when I tried to remove it. After looking at the same cluster in Chrome it looked different, there was a time selector pull down and also the option to remove the wake up schedule.

**What you expected to happen**:
removing wake up schedule should also work in Safari 

**How to reproduce it (as minimally and precisely as possible)**:
login with Safari, try to maintain a wake up schedule for hiobernation, see you can't remove it.

**Anything else we need to know**:

**Environment**:
canary

**Hibernation config in Chrome:** <img width="903" alt="Screenshot 2023-02-16 at 08 38 47" src="https://user-images.githubusercontent.com/32126473/219299205-dd5973a2-4c18-4bc5-8b7f-cdbc3e54c7b0.png">

**Hibernation schedule in Safari:** <img width="901" alt="Screenshot 2023-02-16 at 08 39 27" src="https://user-images.githubusercontent.com/32126473/219299331-b2a22f75-5bc6-4aac-a902-1444f47e4bfa.png">

**Comment by petersutter** [comment]
I could reproduce it, it looks like a rendering issue with the vuetify library that we use. We also have an issue to upgrade to the latest vuetify version as we are still on an older version https://github.com/gardener/dashboard/issues/1418

In my test I cleared the `Wake up at` time and entered a `Hibernate at` time. In this case it looks as if I had set the `Wake up at` time (because of the rendering issue).
![Screenshot 2023-02-16 at 08 59 02](https://user-images.githubusercontent.com/5526658/219303527-201c0534-d2c0-4666-bfea-17acea098186.png)

Once I clear the time it shows the error message that at least one time should be set, so this confirms that it's a rendering issue
![Screenshot 2023-02-16 at 08 59 09](https://user-images.githubusercontent.com/5526658/219303539-cabd96cd-f87b-4527-83b3-9f342ba21ec1.png)

**Comment by grolu** [comment]
Yes. I could also confirm this. I currently have no solution for this. We hope that it will be fixed in an upcoming version of vuetify. I will put this in the icebox as long as we are not on the current major vuetify version.

/hold until we upgraded to vuetify3 #1418

**Comment by grolu** [comment]
As we are now using the latest vuetify version and this is still not fixed, I created an issue for this in the vuetify repository: https://github.com/vuetifyjs/vuetify/issues/18147
